### DISCUSSION TITLE
(Filament v3 Laravel 10) Need help on how can I pass multiple values to a Filament resource from a different table

### DISCUSSION THREAD

**Discussion by chris-john19** [filamentphp/filament#9542] [Help]
Hi!
So I have 2 tables (aside from the default User table)
Namely TableA and TableB

TableA
-ID
-column1A
-column2A
-column3A

TableB
-ID
-column1B
-column2B
-column3B
-column4B
-column5B

My filament Resource is for TableB
where the values for column1B is a select::make based on the values of column1A
after that, the column2B should fetch the corresponding value for column2B based on the selected column1B value. This behavior of column2B will also be carried over to column3B respectively.

The remaining columns of TableB are not related to TableA.

can anyone please help me since I tried using thw documentation on "category" and "subcategory" and it does not populate the values for column2B and column3B respectively.

**Comment by joshembling** [comment]
What you are describing is almost an exact example of this from the docs: 

https://filamentphp.com/docs/3.x/forms/advanced#generating-a-slug-from-a-title

**Reply by chris-john19** [reply]
thank you for your response... I also did the same thing before I posted here (and hate to say it but it doesn't work).
I already did the same thing including this https://filamentphp.com/docs/3.x/forms/advanced#dependant-select-options

Problem is it does not fetch/auto populate the "column2B" when "column2A" already has a selected data. 

I'm on the TableB-resource if that makes any difference.

**Reply by chris-john19** [reply]
I see...
actually I'm not doing a type in column2A .. its a fetch from column1A data that's already there thats why its a Select:: .
